### PR TITLE
Fixes #77: Implemented column headers in table

### LINKED ISSUE #77: [feature request] add column header
when we run `ps` without any args, its output includes column header like this:
~~~
# ps
  PID TTY          TIME CMD
22671 pts/0    00:00:00 bash
23572 pts/0    00:00:00 ps
~~~

currently gops does not have column header, those newcomer users may not know meanings of  1st and 2nd column

### PR DESCRIPTION
**Pull request by yashLadha**
Screenshot 2019-04-23 at 2 13 03 PM

@rakyll

### CODE REVIEW

**Comment by yashLadha** [comment]
@rakyll Can you please look into this PR. I think it is GTG 😕

File: main.go [hunk 1]
```diff
@@ -106,29 +106,48 @@ func processes() {
...
 		fmt.Fprint(buf, p.Path)
 		fmt.Fprintln(buf)
 		buf.WriteTo(os.Stdout)
 	}
 }
 
+func printColumnHeaders() {
+	buf := bytes.NewBuffer(nil)
+	fmt.Fprint(buf, "maxPID")
+	fmt.Fprint(buf, "\t")
```
Comment (rakyll) [inline main.go]: The column in the output are not tab separated.

File: main.go [hunk 2]
```diff
@@ -106,29 +106,48 @@ func processes() {
...
+		fmt.Fprint(buf, "\t")
 		fmt.Fprint(buf, p.Path)
 		fmt.Fprintln(buf)
 		buf.WriteTo(os.Stdout)
 	}
 }
 
+func printColumnHeaders() {
+	buf := bytes.NewBuffer(nil)
+	fmt.Fprint(buf, "maxPID")
```
Comment (rakyll) [inline main.go]: Camel case is probably not the best formatting style we can use here.

Comment (yashLadha) [reply main.go]: So what according to you should be the formatting style?

**Comment by yashLadha** [comment]
Is the issue relevant @rakyll ? We can have a discussion over this.

**Comment by rakyll** [comment]
I think we shouldn't break the output. Although not printed, the columns are self descriptive enough.

**Pull request closed without merging**